packages/embark-whisper/src/index.js

```diff
@@ -149,6 +162,12 @@ class Whisper {
       port: connection.port || '8546',
       type: connection.type || 'ws'
     };
+
+    // TODO: fix storage to also use addProviderInit
+    // execute code called addProviderInit WHEN contracts have been deployed etc..
+    // TODO: diff between addConsoleProviderInit and addProviderInit
+    // or maybe best way is todo everything in the module
```

Comment:
This still needs to be done?

Reply:
Either these comments were moved from somewhere else or @iurimatias introduced them while trying to sort things out, so he'll probably have the most insight to offer.